instructions.html

```diff
@@ -1,72 +1,198 @@
 <html>
 <title>Setup Instructions for the Google API Client Library for Java</title>
 <body>
-<h2>Setup Instructions for the Google API Client Library for Java</h2>
+<h1>Google APIs Client Library for Java</h1>
```

Comment:
The readme.md content doesn't really seem like a good fit for an instructions page...